File: Java/Program/Shirt.java
/*Saya Muhammad Rifky Afandi dengan NIM 2202346 mengerjakan LP2 dalam mata kuliah 
Desain Pemrograman Berbasis Objek untuk keberkahanNya maka saya tidak 
melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.*/


public class Shirt extends Clothing {
    //private attribute
    private String colour;
    private String sleeve_type;

    //Constructor without parameter
    public Shirt(){
        this.colour = "";
        this.sleeve_type = "";
    }

    //constructor with parameter
    public Shirt(String colour, String sleeve_type){
        this.colour = colour;
        this.sleeve_type = sleeve_type;
    }

    /*Getter dan Setter */
    public void setColour(String colour){
        this.colour = colour;
    }

    public String getColour(){
        return this.colour;
    }

    public void setSleeveType(String sleeve_type){
        this.sleeve_type = sleeve_type;
    }

    public String getSleeveType(){
        return this.sleeve_type;
    }
    
}

File: PHP/Program/Shirt.php
<!-- Saya Muhammad Rifky Afandi dengan NIM 2202346 mengerjakan LP2 dalam mata kuliah 
Desain Pemrograman Berbasis Objek untuk keberkahanNya maka saya tidak 
melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin. 
-->

<?php

require('Clothing.php');

class Shirt extends Clothing{
    private $colour = '';
    private $sleeveType = '';

    public function __construct($colour = '', $sleeveType = ''){
        $this->colour = $colour;
        $this->sleeveType = $sleeveType;
    }

    public function setColour($colour){
        $this->colour = $colour;
    }

    public function getColour(){
        return$this->colour;
    }

    public function setSleeve($sleeveType){
        $this->sleeveType = $sleeveType;
    }

    public function getSleeve(){
        return $this->sleeveType;
    }


}




?>